stable/aws-vpc-cni/README.md

```diff
@@ -0,0 +1,62 @@
+# AWS VPC CNI
+
+This chart installs the AWS CNI Daemonset: https://github.com/aws/amazon-vpc-cni-k8s
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+
+## Installing the Chart
+
+First add the EKS repository to Helm:
+
+```shell
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+To install the chart with the release name `aws-node` and default configuration:
```

Comment:
Should we use `aws-vpc-cni` as the chart name/install name as well?  As long as the daemonset is called aws-node everything should be ok... @mogren thoughts?

Reply:
I agree that `aws-vpc-cni` is a better name, but I think people are too used to `aws-node` by now. Would it be confusing if the daemonset has a different name?

Reply:
This is just the release name. I chose `aws-node` as an example in the README but it can be anything and with the default configuration the resources will be named `aws-node*` regardless due to `fullnameOverride` and `nameOverride` in `values.yaml`.

It's unconventional to set `fullnameOverride` and `nameOverride` by default but I think it makes sense given the current situation.

Other option is we rename the whole chart to `aws-node` and leave the name override options blank as is convention. But `aws-node` is a not a great name...

Up to you how you want it 🙂

Reply:
Let's call the chart `aws-vpc-cni` please. As @nckturner mentioned, as long as the Daemonset remains `aws-node`, we're good.

Reply:
It is named that: https://github.com/aws/eks-charts/pull/27/files#diff-d8c9f0ffa17403764b45ca33421e5d4dR2

Reply:
Sorry, I meant the release name, not the chart name.

Reply:
OK cool. Done! To be clear, this is cosmetic though. Users can choose their own name for the release.